projects/1.20-fabric/assets/mythic-charms/mythic_charms/lang/zh_cn.json

```
@@ -0,0 +1,191 @@
+	"text.config.mythic-charms-config.option.charmsEnabled.CharmMountainsStrengthEnabled": "山峦之力护符已启用",
+	"text.config.mythic-charms-config.option.charmsEnabled.CharmSafeTerritoryEnabled": "安全领土护符已启用",
+	"text.config.mythic-charms-config.option.charmsEnabled.CharmQuietPresenceEnabled": "阒然无声护符已启用",
+
+	"text.config.mythic-charms-config.category.charmCombinationsEnabled": "护符组合已启用",
+	"text.config.mythic-charms-config.option.charmCombinationsEnabled.CharmCombinationFeatheredGraceAndHighBoundsEnabled": "启用护符组合：优雅之羽-高界",
+	"text.config.mythic-charms-config.option.charmCombinationsEnabled.CharmCombinationBlazingEmbraceAndBattleFuryEnabled": "启用护符组合：炽热拥抱-狂战士",
+	"text.config.mythic-charms-config.option.charmCombinationsEnabled.CharmCombinationEarthsOrderAndBlazingEmbraceEnabled": "启用护符组合：地球意志-炽热拥抱",
+	"text.config.mythic-charms-config.option.charmCombinationsEnabled.CharmCombinationEarthsOrderAndDrownedFreedomEnabled": "启用护符组合：地球意志-自由水漫",
+	"text.config.mythic-charms-config.option.charmCombinationsEnabled.CharmCombinationEarthsOrderAndWeightlessFlowEnabled": "启用护符组合：地球意志-失重之流",
+	"text.config.mythic-charms-config.option.charmCombinationsEnabled.CharmCombinationEarthsOrderAndBattleFuryEnabled": "启用护符组合：地球意志-狂战士",
+	"text.config.mythic-charms-config.option.charmCombinationsEnabled.CharmCombinationBotanicBlessingAndFeatheredGraceEnabled": "启用护符组合：植物祝福-优雅之羽",
+	"text.config.mythic-charms-config.option.charmCombinationsEnabled.CharmCombinationFleetingStridesAndHighBoundsEnabled": "启用护符组合：轻快步伐-高界",
+	"text.config.mythic-charms-config.option.charmCombinationsEnabled.CharmCombinationFleetingStridesAndBattleFuryEnabled": "启用护符组合：轻快步伐-狂战士",
+	"text.config.mythic-charms-config.option.charmCombinationsEnabled.CharmCombinationWeightlessFlowAndFeatheredGraceEnabled": "启用护符组合：失重之流-优雅之羽",
+	"text.config.mythic-charms-config.option.charmCombinationsEnabled.CharmCombinationClimbersPathAndHighBoundsEnabled": "启用护符组合：攀山之径-高界",
+	"text.config.mythic-charms-config.option.charmCombinationsEnabled.CharmCombinationEchoingWrathAndBlazingEmbraceEnabled": "启用护符组合：狂怒回响-炽热拥抱",
+	"text.config.mythic-charms-config.option.charmCombinationsEnabled.CharmCombinationEchoingWrathAndBattleFuryEnabled": "启用护符组合：狂怒回响-狂战士",
+	"text.config.mythic-charms-config.option.charmCombinationsEnabled.CharmCombinationQuietPresenceAndFeatheredGraceEnabled": "启用护符组合：阒然无声-优雅之羽",
+
+	"text.config.mythic-charms-config.option.experienceNuggetMobCap": "周围的小怪数量，超过该数量将不再掉落经验金块",
+
+	"advancements.mythic_charms.story.root.title": "神话护符",
+	"advancements.mythic_charms.story.root.desc": "万物尽在掌握",
+	"advancements.mythic_charms.story.resonance_table.title": "漂亮的小桌子",
+	"advancements.mythic_charms.story.resonance_table.desc": "制作一个共鸣桌",
+	"advancements.mythic_charms.story.fragile_charm_base.title": "打好基础",
+	"advancements.mythic_charms.story.fragile_charm_base.desc": "制作一个脆弱的护符底座",
+	"advancements.mythic_charms.story.first_charm.title": "《护符的魅力》101节",

```

```suggestion
	"advancements.mythic_charms.story.first_charm.title": "护符制造101",
```
101一般指“入门教程”，不是书中具体某一节

---

或者可以把101放到书名号里面去（）